webapp/components/interop.js

```diff
@@ -319,32 +390,38 @@ class WPTInterop extends QueryBuilder(
     return labels;
   }
 
-  computeTestRuns(metadata) {
-    return metadata && metadata.test_runs;
-  }
-
-  computeTestPaths(fileMetrics) {
-    const paths = fileMetrics && fileMetrics.map(m => m.dir) || [];
+  computeTestPaths(searchResults) {
+    const paths = searchResults && searchResults.results.map(r => r.test) || [];
     return new Set(paths);
   }
 
-  computeFileMetrics(allMetrics) {
-    let fileMetrics = [];
-    for (const metric of allMetrics.data) {
-      if (this.computePathIsATestFile(metric.dir)) {
-        fileMetrics.push(metric);
+  precomputedInteropLoaded(precomputedInterop) {
+    const searchResults = {
+      runs: this.testRuns,
+      results: [],
+    };
+    if (precomputedInterop) {
+      for (const metric of precomputedInterop.data) {
+        if (this.computePathIsATestFile(metric.dir)) {
```

Comment:
This line shows that calling it `dir` in the format is a bit odd. Is it to match the old format, or can it be called `path` instead?

Reply:
dir is the existing precomputed interop's field name, because we (redundantly) group by each directory as well as test-file. searchcache's results calls it `path`.